packages/webapp/.env

```diff
@@ -42,13 +42,13 @@ NEXT_PUBLIC_ELECTION_MEETING_DURATION_MS = "2400000"
 NEXT_PUBLIC_ELECTION_COMMUNITY_ROOM_URL = "[NOT SET]"
 
 # BOX: SUBCHAIN
-NEXT_PUBLIC_SUBCHAIN_WASM_URL = "http://localhost:3032/v1/subchain/eden-micro-chain.wasm"
-NEXT_PUBLIC_SUBCHAIN_STATE_URL = "http://localhost:3032/v1/subchain/state"
-NEXT_PUBLIC_SUBCHAIN_WS_URL = "ws://localhost:3032/v1/subchain/eden-microchain"
+NEXT_PUBLIC_SUBCHAIN_WASM_URL = "https://eden-box.edenia.cloud/v1/subchain/eden-micro-chain.wasm"
+NEXT_PUBLIC_SUBCHAIN_STATE_URL = "https://eden-box.edenia.cloud/v1/subchain/state"
+NEXT_PUBLIC_SUBCHAIN_WS_URL = "ws://eden-box.edenia.cloud/v1/subchain/eden-microchain"
 NEXT_PUBLIC_SUBCHAIN_SLOW_MO = "false"
 
 # BOX: IPFS
-NEXT_PUBLIC_BOX_ADDRESS = "http://localhost:3032"
+NEXT_PUBLIC_BOX_ADDRESS = "https://eden-box.edenia.cloud"
```

Comment:
I think we can leave these values as they were before to let people run them completely locally.

Reply:
I maked the changes, for replace de .env in the action GitHub.